tests/mount-layer.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "mount-layer" {
+	# Create a layer.
+	run storage --debug=false create-layer
+	[ "$status" -eq 0 ]
+	[ "$output" != "" ]
+	layer="$output"
+
+	# Mount the layer.
+	run storage --debug=false mount $layer
+	[ "$status" -eq 0 ]
+	[ "$output" != "" ]
+	# Check it layer is mounted
+	run storage --debug=false mounted $layer
+	[ "$status" -eq 0 ]
+	[ "$output" != "" ]
```

Comment:
Can we compare to `"$layer mounted"` here?

Reply:
Fixed